src/ingress/forwarder.rs

```diff
@@ -174,16 +174,16 @@ pub struct PeerHandle {
     /// Peer info.
     pub info: builderhub::Peer,
     /// Sender to the peer forwarder.
-    pub sender: pchannel::UnboundedSender<Arc<ForwardingRequest>>,
+    pub sender: channel::UnboundedSender<Arc<ForwardingRequest>>,
 }
 
 pub fn spawn_forwarder(
     name: String,
     url: String,
     client: reqwest::Client, // request client to be reused for http senders
     task_executor: &TaskExecutor,
-) -> eyre::Result<pchannel::UnboundedSender<Arc<ForwardingRequest>>> {
-    let (request_tx, request_rx) = pchannel::unbounded_channel();
+) -> eyre::Result<channel::UnboundedSender<Arc<ForwardingRequest>>> {
```

Comment:
`channel` here is quite opaque (I have to over or jump to remember it is a priority queue), I liked `pchannel`. Alternatively, use a fully qualified `priority::channel`.